webserver/webserver/src/main/java/io/helidon/webserver/ServerListener.java

```diff
@@ -173,6 +168,15 @@ class ServerListener implements ListenerContext {
         ith.start();
     }
 
+    private void initServerThread() {
```

Comment:
closeFuture and serverThread are not volatile, yet instances are created from different threads